### ISSUE TITLE
.gitignore Failing to Ignore

### ISSUE THREAD

**Issue by maximilianfanning** [maximilianfanning/NameGenerator#6]
The .gitignore file currently will not ignore a .cache file in the Debug folder within obj. 
.cache, Debug and obj all have ignore commands within the file. 
Why is this happening and how can we solve this issue?

**Comment by maximilianfanning** [comment]
To be clear, git is ignoring other .cache files within the same folder. It seems to be case sensitive. I tried using ignore commands for different categories that would affect this file, but nothing seemed to work.

**Comment by michaelcfanning** [comment]
This seems really curious to me. Do we have a clear repro? i.e., if I enlist in master and build, will I see it?

**Comment by maximilianfanning** [comment]
Alright, @michaelcfanning! I do not know what I did, but I went on the internet to search for our problem. I found a couple .gitignore samples and copied the parts we needed. One of them fixed it, but then it said the branch was even with main. I do not know what happened, but it's good! I know you're going to want to see what I am talking about, though, so I will not close this issue just yet.
